src/main/java/core/basesyntax/OperationStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private final Map<FruitTransaction.Operation, OperationHandler> operationHandlers;
```

Comment:
According to the encapsulation principle, consider setting the visibility of 'operationHandlers' to private if it's not intended to be accessed from outside the class.